lib/urlwatch/storage.py

```diff
@@ -126,6 +126,8 @@
             'enabled': False,
             'bot_token': '',
             'chat_id': '',
+            'monospaced': False,
```

Comment:
Use `monospace` for this key name, not `monospaced`.

Reply:
renamed